Specify that scenarios run in parallel by default (#561)
grafana/k6-docs: src/data/markdown/translated-guides/en/02 Using k6/14 Scenarios.md

- Every scenario can use a distinct VU and iteration scheduling pattern,
  powered by a purpose-built [executor](#executors). This enables modeling
  of advanced execution patterns which can better simulate real-world traffic.

> - They run independently, which means that they also by default run in parallel. But can be made somewhat sequential through setting `startTime` accordingly.

Reviewer: ```suggestion
- They are independent from each other and run in parallel, though can be made to appear sequential through the `startTime` property.
```
